superset/migrations/versions/f6196627326f_update_chart_permissions.py

```diff
@@ -36,7 +36,7 @@
 
 # revision identifiers, used by Alembic.
 revision = "f6196627326f"
-down_revision = "143b6f2815da"
+down_revision = "ae1ed299413b"
```

Comment:
```suggestion
down_revision = "143b6f2815da"
```

We usually can't change the `down_revision`, otherwise someone who might've run the migration already will end up in a bad state.